pkg/types/openstack/validation/platform_test.go

```diff
@@ -67,7 +67,7 @@ func TestValidatePlatform(t *testing.T) {
 				return p
 			}(),
 			networking: validNetworking(),
-			valid:      false,
+			valid:      true,
```

Comment:
This only check the case where both `externalNetwork` and `lbFloatingIP` are unset.
Would be good to have unit tests covering the other cases too, where just one of `lbFloatingIP` or `externalNetwork` is set (invalid), or where both are set (valid). IIUC we do not treat these cases and this goes against the above documentation change.

Reply:
We are postponing the additional tests and validations until the validations code becomes stable again. Otherwise it will be a pain to rebase.

Reply:
Works for me, as long as we do not forget about it. Could you create a Jira card for it?

Reply:
Created https://issues.redhat.com/browse/OSASINFRA-979

We can continue discussion there.